README.md

```diff
@@ -149,16 +149,21 @@ Community
 
 Localization and translations are handled by [Transifex][tx], which is used by all Cozy's apps.
 
-As a _translator_, you can login to [Transifex][tx-signin] (using your Github account) and claim an access to the [app repository][tx-app]. Locales are pulled when app is build before publishing.
+As a _translator_, you can login to [Transifex][tx-signin] (using your Github account) and claim access to the [app repository][tx-app]. Locales are pulled [by the pipeline][yarn tx in travis.yml] when app is build before publishing.
 
-As a _developer_, you must [configure the transifex client][tx-client], and claim an access as _maintainer_ to the [app repository][tx-app]. Then please **only update** the source locale file (usually `en.json` in client and/or server parts), and push it to Transifex repository using the `tx push -s` command.
+As a _developer_, you must [configure the transifex client][tx-client], and claim access as _maintainer_ to the [app repository][tx-app]. Then please **only update** the source locale file (usually `en.json` in client and/or server parts), and push it to Transifex repository using the `tx push -s` command.
 
+###### Drive specificity
+
+CozyDrive and CozyPhotos share this mono-repository but the translations are split into two projects Transifex.
+That is why the translation files are not present in this repository.
+
+To update the translation in other language, make the changes directly on [tx-translate].
```

Comment:
Your links redirects to the `fr` translation, do we have a "more global" link?

Reply:
I found a solution to make it more global